packages/badge/docs/overview.mdx

```diff
@@ -52,3 +52,43 @@ A small decorative indicator used to call attention to an item, such as an unrea
 	<IndicatorDot tone="action" />
 </Flex>
 ```
+
+An IndicatorDot should not be used on its own, as it gives no context. It should instead be considered as part of a wider composition.
+
+```jsx live
+<Flex flexDirection="column" as="ul">
+	<Flex
+		as="li"
+		aria-selected="false"
+		borderColor="muted"
+		borderY
+		css={{
+			backgroundColor: boxPalette.systemInfoMuted,
+		}}
+		flexDirection="column"
+		gap={0.5}
+		justifyContent="space-between"
+		padding={1.5}
+		width="100%"
+	>
+		<Flex justifyContent="space-between">
+			<Text fontSize="md" fontWeight="bold">
+				Action required
+			</Text>
+
+			<Flex alignItems="center" gap={0.5}>
+				<Text color="muted">10:15am</Text>
+				<IndicatorDot tone="action" />
+				<VisuallyHidden>Unread message</VisuallyHidden>
```

Comment:
I wonder if it would be better off moving this to the start of the `li`, so it reads out `Unread message, Action required ....`?

Reply:
the issue with putting the badge first is visually it looks like a bullet point